
mocap2ublox: {

ublox_frequency: 5.0, #hz

global_horizontal_accuracy: 0.4, #standard deviations
global_vertical_accuracy: 0.6,
global_speed_accuracy: 0.4,
relative_horizontal_accuracy: 0.02,
relative_vertical_accuracy: 0.06,
relative_speed_accuracy: 0.02,

noise_on: True,

ref_lla: [40.267320, -111.635629, 1387.0],

sigma_rover_pos: 0.5, #TODO: tune this
sigma_rover_vel: 0.5, #TODO: tune this
sigma_rover_relpos: 0.0, #TODO: tune this
sigma_base_pos: 0.0, #TODO: tune this
sigma_base_vel: 0.0, #TODO: tune this

lpf_on: False, #low pass filter of measurements

A: 6378137.0,       # WGS-84 Earth semimajor axis (m), from ekf_ros.cpp
B: 6356752.314245,  # Derived Earth semiminor axis (m), from ekf_ros.cpp

}